### DISCUSSION TITLE
Updating the "how to add vlan segmentation for homekit iot devices with unifi"

### DISCUSSION THREAD

**Discussion by interSAP** [RobertDWhite/WhiteMatterTech#18] [Ideas]
Hello, 

I've been experimenting with this post: https://whitematter.tech/posts/how-to-add-vlan-segmentation-for-homekit-iot-devices-with-unifi/ on my own unifi setup, and one of the challenges that I find is that yours looks quite different than mine, probably due to using a newer version of the unifi controller (8.11.3 as of today). 

One of the main differences is in the firewall rules. I'm trying to wrap up my mind on where some of the default rules are coming from, as you don't seem to have them in your article. 

So, I was wondering if it makes sense to refresh the post on a more current setup.

Thanks,

**Comment by RobertDWhite** [comment]
Hi @interSAP
It definitely makes sense to update the images. I plan to do that at some point, hopefully in the near future.

Thank you!
